clientupdate/clientupdate.go

```diff
@@ -175,6 +175,10 @@ func (up *Updater) getUpdateFunction() (fn updateFunction, canAutoUpdate bool) {
 			return up.updateLinuxBinary, true
 		case distro.Alpine:
 			return up.updateAlpineLike, true
+		case distro.Unraid:
+			// Unraid runs from memory, updates must be installed via the Unraid
+			// plugin manager to be persistent.
```

Comment:
Per https://forums.unraid.net/topic/72240-solved-is-there-a-way-to-installuninstall-plugins-from-script/#comment-676870 there's a `plugin` CLI which we may be able to use here to perform the update.

```suggestion
			// plugin manager to be persistent.
			// TODO(awly): implement Unraid updates using the 'plugin' CLI.
```